viewer/src/datamodels/cad/styling/ByAssetNodeSet.ts

```diff
@@ -90,4 +100,12 @@ export class ByAssetNodeSet extends NodeSet {
   getIndexSet(): IndexSet {
     return this._indexSet;
   }
+
+  /* @internal */
+  Serialize(): SerializedNodeSet {
+    return {
+      token: this.classToken,
+      state: cloneDeep(this._filter)
```

Comment:
Will this crash and burn (as in, result in random field names etc) in minified code @maksnester?

Reply:
I believe this is fine, since it works in docs. Where reveal is minified.